index.html

```diff
@@ -4,36 +4,115 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <title>TripHouse</title>
+    <link rel="icon" href="./images/favicons/favicon.ico" />
     <link rel="stylesheet" href="styles/style.css" />
   </head>
   <body class="body">
+    <div class="top-section">
+      <div class="img-top">
```

Comment:
"img-top"
1.bad classname - you use it like container for something 
2.he have almost same parameters with top-section